### PR TITLE
Add tool: task/enhancement with more than 3 regressions

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: auto_nag/scripts/task_enhancement_regression.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+class TaskEnhancementRegression(BzCleaner):
+    def __init__(self):
+        super(TaskEnhancementRegression, self).__init__()
+        self.threshold = self.get_config("threshold", 3)
+        self.regressors = {}
+
+    def description(self):
+        return "Task or enhancement which caused regression"
```
Comment (marco-c) [inline auto_nag/scripts/task_enhancement_regression.py]: Nit: `f"Task or enhancement which caused more than {self.threshold} regressions"`

File: auto_nag/scripts/task_enhancement_regression.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    def columns(self):
+        return ["id", "summary", "product", "component"]
+
+    def handle_bug(self, bug, data):
+        if len(bug["regressions"]) < self.threshold:
+            return None
+
+        return bug
+
+    def get_bz_params(self, date):
```
Comment (marco-c) [inline auto_nag/scripts/task_enhancement_regression.py]: Maybe we should limit the query to bugs which have been touched recently?

**Comment by sylvestre** [comment]
How this is different from https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/scripts/warn_regressed_by.py ?

**Comment by marco-c** [comment]
> How this is different from https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/scripts/warn_regressed_by.py ?

The only difference is that warn_regressed_by only warns about 'defect' bugs that caused more than three regressions.

**Comment by marco-c** [comment]
> The only difference is that warn_regressed_by only warns about 'defect' bugs that caused more than three regressions.

Actually no, it warns about any bug of any type that caused more than three regressions (assuming the regressions were all filed recently enough).

Let's close this as it's almost redundant.

**Pull request closed without merging**